### PR TITLE
Dynamic channel 2

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
based on #1416

File: templates/channels/types/africastalking/claim.haml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+-extends "channels/channel_claim_form.html"
```
Comment (rowanseymour) [inline templates/channels/types/africastalking/claim.haml]: I don't think any of the channels I ported use this file - should they?

Comment (norkans7) [reply templates/channels/types/africastalking/claim.haml]: that file extends 'smartmin/form.html' with some styles so I think it is similar

Comment (rowanseymour) [reply templates/channels/types/africastalking/claim.haml]: I'm just checking we're being consistent - maybe all channel claim pages should extend this

**Pull request merged**